plugin/jvm/src/main/java/com/twosigma/beaker/table/serializer/TableDisplaySerializer.java

```diff
@@ -58,6 +58,7 @@ public void serialize(TableDisplay value,
       }
       jgen.writeBooleanField("headersVertical", value.getHeadersVertical());
       jgen.writeObjectField("values", value.getValues());
+      jgen.writeObjectField("hasMapValues", value.isHasMapValues());
```

Comment:
yes, this is a flag for javascript to process only arrays with values as objects, so the part of the work is done on server. This is some kind of optimization.